src/ado/reprun.ado

```diff
@@ -625,6 +646,9 @@ end
 
     local prev_line1 ""
     local prev_line2 ""
+	
+    * Local for empty tables
+    local any_lines_written 0	
```

Comment:
You might be able to find a way to reset this when "stepping back"; alternatively, you might _not_ want to have this show up until each file is closed for the last time. This may or may not be predictable behavior.

I would suggest, since we will know what line the file has called a sub-do-file at, to report "no differences/errors in lines XX-YY" (don't worry about the exact message wording yet). I think we should be able to pick up XX/YY? @kbjarkefur?

Reply:
I have not been able to figure this out. If there are no changes in the main file at all, then it does display "no mismatches...". But I have not been able to figure out how to reset it when "stepping back".  

Can think about this more after working on other things for the conference.